WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -98,7 +98,7 @@
 
         <StackPanel Spacing="10">
           <c:PreviewItem Icon="{StaticResource timer_regular}"
-                         Text="miners will work hard to confirm your transaction within">
+                         Text="your transaction should be confirmed within">
```

Comment:
not sure if `should` is the right word here. I won't to convey that this is never an accurate time estimate.

Reply:
Can we use `could` ?

Reply:
https://english.stackexchange.com/questions/30629/whats-the-difference-between-should-and-could-in-the-present-tense